src/node/mini_miner.cpp

```diff
@@ -125,6 +125,48 @@ MiniMiner::MiniMiner(const CTxMemPool& mempool, const std::vector<COutPoint>& ou
     SanityCheck();
 }
 
+MiniMiner::MiniMiner(const std::vector<MiniMinerMempoolEntry>& manual_entries,
+                     const std::map<Txid, std::set<Txid>>& descendant_caches)
```

Comment:
leaving it as is :sweat_smile:

Reply:
I also found `manual_entries` vs `m_entries` make me check back a couple times. My understanding is that the `manual_entries` are just fully specified rather than read from the mempool. Would maybe `manually_submitted_entries` fit the bill?